pkg/cri/server/service.go

```diff
@@ -129,6 +132,10 @@ func NewCRIService(config criconfig.Config, client *containerd.Client) (CRIServi
 		initialized:        atomic.NewBool(false),
 	}
 
+	if ns != nil {
```

Comment:
I don't think there is a case in the code base where ns would be nil.. and if it was and we continued I think you'd throw an exception now if pod networking was on..  Suggest making this an error out if nil.. or maybe add a config bool for turning off these new metrics to overcome the exception if they are not properly initialized.

Reply:
I think I initially did this because I saw a `NoPrometheus` option somewhere and figured I might need to handle that... but alas that's just for the cgroup metrics, which indeed could bloat prom and some people might not want to do that. I don't think these new metrics need to be optional.